internal/services/cognitive/cognitive_deployment_resource.go

```diff
@@ -146,11 +146,10 @@ func (r CognitiveDeploymentResource) Arguments() map[string]*pluginsdk.Schema {
 						ForceNew: true,
 					},
 					"capacity": {
-						Type:         pluginsdk.TypeInt,
-						Optional:     true,
-						ForceNew:     true,
-						Default:      1,
-						ValidateFunc: validation.IntBetween(1, 10000),
```

Comment:
actually shouldn't this be validating that it is greater then 0 and not allow negative values?

Reply:
Sure, let me update this one